### ISSUE TITLE
nonExpandable behaviour

### ISSUE THREAD

**Issue by vinzentt** [react-bootstrap-table/react-bootstrap-table2#697]
Hi all, 

I'm trying to use the nonExpandable argument to prevent certain rows from being expanded. My problem is that the nonExpandable is only working with the expandAll header icon.

Here is my code inside storybook

```javascript
import React from 'react';

import BootstrapTable from 'react-bootstrap-table-next';
import Code from 'components/common/code-block';
import { productsExpandRowsGenerator } from 'utils/common';

const products = productsExpandRowsGenerator();

const columns = [{
  dataField: 'id',
  text: 'Product ID'
}, {
  dataField: 'name',
  text: 'Product Name'
}, {
  dataField: 'price',
  text: 'Product Price'
}];

const expandRow = {
  renderer: row => (
    <div>
      <p>{ `This Expand row is belong to rowKey ${row.id}` }</p>
      <p>You can render anything here, also you can add additional data on every row object</p>
      <p>expandRow.renderer callback will pass the origin row object to you</p>
    </div>
  ),
  showExpandColumn: true,
  nonExpandable: [1, 3]
};

export default () => (
  <div>
    <h3>The second and fourth row is not expandable</h3>
    <BootstrapTable
      keyField="id"
      data={ products }
      columns={ columns }
      expandRow={ expandRow }
    />
    <Code>{ sourceCode }</Code>
  </div>
);
```
So here here the expandColumn is still shown with an expand indicator for columns 1 and 3, and when clicking on it, the rows are being expanded. Is this the expected behaviour ?

Thanks for your answer

**Comment by vinzentt** [comment]
So in my opinion, there should not be an expand indicator on nonExpandable rows, and clicking on the expand indicator cell should have no effect. I did this PR https://github.com/react-bootstrap-table/react-bootstrap-table2/pull/698 to achieve this. Tell me if it needs some modifications.

Thanks

**Comment by AllenFang** [comment]
@vinzentt I will check this out, thanks!

**Comment by AllenFang** [comment]
@vinzentt fixed in `react-bootstrap-table-next@1.4.3`

https://react-bootstrap-table.github.io/react-bootstrap-table2/blog/2018/12/21/version-bump.html
